backend/src/main/java/com/soen341/instagram/utils/AccountVerifier.java

```diff
@@ -52,4 +53,12 @@ public static void checkNameFormat(final String name)
 			throw new InvalidNameException();
 		}
 	}
+
+	public static void checkIfSamePassword(final String oldPassword, final String newPassword)
```

Comment:
This method will always fail because the `oldPassword` will be an encrypted string while the new password will be a normal string. If I recall, once a password is encoded, it is impossible to decode it. So, I'm not sure how we would handle this exception tbh.

Reply:
The PasswordEncoder (https://docs.spring.io/spring-security/site/docs/4.2.4.RELEASE/apidocs/org/springframework/security/crypto/password/PasswordEncoder.html) and BCryptPasswordEncoder (https://docs.spring.io/spring-security/site/docs/4.2.4.RELEASE/apidocs/org/springframework/security/crypto/password/PasswordEncoder.html), and implementation of the PasswordEncoder interface, have "matches" methods that can allow you to check if I'm not mistaken

Reply:
Thanks, I didn't know this existed and it was exacly what was needed!